src/cmd/deploy.rs

```diff
@@ -72,7 +72,9 @@ pub fn run(args: &Args) -> Result<()> {
             &format!(
                 r###"
 cros-workon-{board} start {packages_str}
-~/trunk/src/scripts/update_kernel.sh {} --remote={} --ssh_port {} --remote_bootargs
+TOPDIR=~/trunk
+[ -d $TOPDIR ] || TOPDIR=~/chromiumos
```

Comment:
It would be nice if you add a comment that says which CL introduced the dir change (if not and if it seems to take some time to lookup, you can just ignore this comment!)

Reply:
Yeah, https://chromium-review.googlesource.com/c/chromiumos/chromite/+/4835006 removes `~/trunk`.